src/clr-angular/layout/tabs/tab-link.directive.spec.ts

```diff
@@ -13,19 +13,21 @@ import { TabsService } from './providers/tabs.service';
 import { ClrTabLink } from './tab-link.directive';
 import { TABS_ID_PROVIDER } from './tabs-id.provider';
 import { ClrTabsModule } from './tabs.module';
+import { TabsLayout } from '@clr/angular';
```

Comment:
Wrong path

Reply:
oops! fixed.
should I also change it to simple string, or using the import (correct one) in unit tests is ok?